src/server.js

```diff
@@ -196,10 +199,11 @@ const config = new Config()
  * @description Starts the web server
  */
 async function start () {
+  const cache = new Cache()
```

Comment:
I can pass a singular cache to the db and for api request caching but have kept  them seperate for now. Any thoughts on a singular cache for everything?